src/common/options.h

```diff
@@ -36,7 +36,7 @@ struct Option {
     case TYPE_ADDR: return "entity_addr_t";
     case TYPE_ADDRVEC: return "entity_addrvec_t";
     case TYPE_UUID: return "uuid_d";
-    case TYPE_SIZE: return "size_t";
+    case TYPE_SIZE: return "uint64_t";
```

Comment:
i think we should drop this function, as it is not used anywhere.

Reply:
I have not removed it as I did not know its use (may be some obscure use which is not immediately apparent) and just modified it to be in line with the rest of the patch. If it is not really used and therefore should be dropped I guess it should be a separate PR for clarity.

Reply:
> If it is not really used and therefore should be dropped I guess it should be a separate PR for clarity.

agreed. it's not in the scope of this PR. i will create a PR to do this.